examples/npm-workspaces/client/src/index.ts

```diff
@@ -0,0 +1,57 @@
+/**
+ * This is the client-side code that uses the inferred types from the server
+ */
+import {
+  createTRPCClient,
+  splitLink,
+  unstable_httpBatchStreamLink,
+  unstable_httpSubscriptionLink,
+} from '@trpc/client';
+/**
+ * We only import the `AppRouter` type from the server - this is not available at runtime
+ */
+import type { AppRouter } from '@examples-npm-workspaces/server/src';
```

Comment:
If I understand right you're saying this is resolving 'any', but as best I can tell you haven't told typescript how to find this in the first place. You'll need a path to `@examples-npm-workspaces` in your tsconfig

Reply:
It's resolved via the package.json `@examples-npm-workspaces/server` → npm workspaces.